### PR TITLE
Updated ArgoCD alerts queries.

### PR DESCRIPTION
**Pull request by miransar**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Are all documentation links publically accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ ] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/argocd/ApplicationOutOfSync.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: |+
 
 type: STATIC
 nrql:
-  query: "SELECT count(argocd_app_info) FROM Metric WHERE sync_status = 'OutOfSync' AND job = 'argocd-metrics'"
+  query: "SELECT count(argocd_app_info) FROM Metric WHERE sync_status = 'OutOfSync' WHERE instrumentation.name = 'remote-write'"
```
Comment (josephgregoryii) [inline alert-policies/argocd/ApplicationOutOfSync.yml:8]: I think there might be an accidental change in the syntax here!
```suggestion
  query: "SELECT count(argocd_app_info) FROM Metric WHERE sync_status = 'OutOfSync' AND instrumentation.name = 'remote-write'"
```

**Review by josephgregoryii — COMMENTED** [review]
Looks great - one small comment below

File: alert-policies/argocd/ApplicationOutOfSync.yml (hunk 1 above)
Comment (miransar) [reply alert-policies/argocd/ApplicationOutOfSync.yml]: @josephgregoryii Done. Thanks for pointing that out. 👍

Comment (josephgregoryii) [reply alert-policies/argocd/ApplicationOutOfSync.yml]: No worries! Thanks for fixing that :)

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**